crates/networking/discv5/src/eth2.rs

```diff
@@ -0,0 +1,103 @@
+use alloy_primitives::{B256, aliases::B32, fixed_bytes};
+use ethereum_ssz::Encode;
+use ethereum_ssz_derive::{Decode, Encode};
+use ream_consensus::fork_data::ForkData;
+
+// Constants
+pub const FAR_FUTURE_EPOCH: u64 = 18446744073709551615;
+pub const GENESIS_FORK_VERSION: B32 = fixed_bytes!("0x00000000");
```

Comment:
```suggestion
```

we already have defined these 2 constants please import them instead of redefining them